docs/src/documentation/02-foundation/05-design-tokens/01-general.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: Introduction
+description: Design tokens store visual design attributes. They help us make our UI more consistent and support custom themes.
+---
+
+## What are design tokens?
+
+Orbit design tokens could be considered as a simple way to describe, document
+and use the visual language of Orbit design system.
+They represent our design decisions
+like the value of color, typography attributes, breakpoints, spacings, etc.
+
+## Organization
+
+We organize design tokens based on their planned and possible usage variety.
+Therefore we have two main levels of design tokens:
+**global design tokens** and **component specific design tokens**.
+
+### Global design tokens
+
+Global level of our design tokens represents the core
+and main level of properties that can be used globally,
+in most use cases and describes the majority of the [foundation](/foundation).
```

Comment:
```suggestion
The global level of our design tokens represents the core
and main level of properties that can be used globally,
in most use cases. It also describes the majority of the [foundation](/foundation).
```

Just a suggestion. Don't accept it directly, as it may break linting rules 💭